src/mantine-core/src/components/Spoiler/Spoiler.module.css

```diff
@@ -1,6 +1,8 @@
 .root {
   position: relative;
-  margin-bottom: rem(24px);
+  &[data-has-spoiler] {
+    margin-bottom: rem(24px);
```

Comment:
Properties must not be set inside `&[data-has-spoiler]` – it will be impossible to override without !important. Use css variables in this case, see other components styles as reeference.